turtlebot_navigation/CMakeLists.txt

```diff
@@ -21,6 +21,43 @@ include_directories(${catkin_INCLUDE_DIRS})
 add_executable(laser_footprint_filter src/laser_footprint_filter.cpp)
 target_link_libraries(laser_footprint_filter ${catkin_LIBRARIES})
 
+# Generate camera param files
+include($ENV{ROS_ROOT}/core/rosbuild/rosbuild.cmake)
```

Comment:
This needs a dependency added on rosbuild which I'd rather not add. That's been deprecated for years now. Is the only need for this the `rosbuild_find_ros_package` below?